plugin/storage/badger/spanstore/cache.go

```diff
@@ -18,7 +21,7 @@ type CacheStore struct {
 	// Given the small amount of data these will store, we use the same structure as the memory store
 	cacheLock  sync.Mutex // write heavy - Mutex is faster than RWMutex for writes
 	services   map[string]uint64
-	operations map[string]map[string]uint64
+	operations map[string]map[trace.SpanKind]map[string]uint64
```

Comment:
please add a comment explaining the structure of the map, which is quite complex